test/android/core/javatests/src/org/xwalk/core/xwview/test/SetInitialScaleTest.java

```diff
@@ -75,12 +76,12 @@ public void testSetInitialScale2() throws Throwable {
                 + "<p style='height:" + height + "px;width:" + width + "px'>"
                 + "testSetInitialScale</p></body></html>";
         final float defaultScaleFactor = 0;
-        final float defaultScale = 0.5f;
-        final float scaleFactor = 0.25f;
+        final float defaultScale = getInstrumentation().getTargetContext(
+                ).getResources().getDisplayMetrics().density;
 
         assertEquals(defaultScaleFactor, getScaleFactor(), .01f);
         loadDataSync(null, page, "text/html", false);
-        assertEquals(scaleFactor, getScaleFactor(), .01f);
+        assertEquals(defaultScale, getPixelScale(), .01f);
```

Comment:
Function prototype:
`void assertEquals (float expected, 
                float actual, 
                float delta)`
If the two number's subtraction is less than delta, they are equals.